analyzers/src/SonarAnalyzer.CSharp/Rules/ParameterValidationInMethodWalker.cs

```diff
@@ -44,36 +44,38 @@ internal class ParameterValidationInMethodWalker : SafeCSharpSyntaxWalker
 
         public IEnumerable<Location> ArgumentExceptionLocations => argumentExceptionLocations;
 
-        public ParameterValidationInMethodWalker(SemanticModel semanticModel)
-        {
-            this.semanticModel = semanticModel;
-        }
+        public ParameterValidationInMethodWalker(SemanticModel semanticModel) => this.semanticModel = semanticModel;
```

Comment:
NIT: Our guideline says any non-trivial expression should be on the next line:
```suggestion
        public ParameterValidationInMethodWalker(SemanticModel semanticModel) =>
            this.semanticModel = semanticModel;
```